source/retryable-writes/tests/supported-multi-statement-failure.yml

```diff
@@ -17,15 +17,11 @@ tests:
             name: "bulkWrite"
             arguments:
                 requests:
-                    -
-                        name: "updateOne"
-                        arguments:
-                            filter: { _id: 1 }
-                            update: { $inc: { x : 1 }}
-                    -
-                        name: "insertOne"
-                        arguments:
-                            document: { _id: 2, x: 22 }
+                    - updateOne:
+                        filter: { _id: 1 }
+                        update: { $inc: { x : 1 }}
+                    - insertOne:
+                        document: { _id: 2, x: 22 }
                 options: { ordered: true }
```

Comment:
The [command-monitoring tests](https://github.com/mongodb/specifications/blob/master/source/command-monitoring/tests/bulkWrite.yml#L52-L58) also put "ordered" as a top-level bulkWrite argument. Another quirk of the old format because in the CRUD spec bulkWrite has only two arguments "requests" and "options": `bulkWrite(requests: WriteModel[], options: Optional<BulkWriteOptions>)`

Reply:
I didn't realize you had to do that in https://github.com/mongodb/specifications/pull/206/files#diff-d3ed3efa0100edd60ecfdfe561e64f29R28 and wrongfully assumed this was just about the `requests` structure.

It looks like that was also done for APM's insertMany tests in https://github.com/mongodb/specifications/blob/88e51a7b53346010f50ef596b38e636021ee0462/source/command-monitoring/tests/insertMany.yml#L61.

Despite prior art, that just seems incorrect. The CRUD spec defines multiple options for BulkWriteOptions and InsertManyOptions (`ordered` and `bypassDocumentValidation`). I also find it odd that `writeConcern` is placed within the operation's `arguments` object, as that is purposefully not defined in the spec (drivers decide on their own to support per-operation write concerns).

Regarding `writeConcern`, I have that alongside the `operation` object (see: [`unsupported-write-concern.yml`](https://github.com/mongodb/specifications/blob/88e51a7b53346010f50ef596b38e636021ee0462/source/retryable-writes/tests/unsupported-write-concern.yml#L11)) and specifically mention it can be specified on the collection or operation as supported by each driver.

Reply:
@ajdavis: Is this perhaps reason enough to look at updating the APM tests or just allowing the formats to diverge (so drivers don't need to rewrite their APM test runners)?